affiliates/base/static/js/global.js

```diff
@@ -174,6 +174,13 @@
     });
 
 
+    // Firefox Friends Messaging Modal For HP - Launches On Load If Exists
+    if ($('#friends-msg').length) {
```

Comment:
It's a micro-optimization, but you could just fetch `#friends-msg` once. You could even be tricky and rely on the fact that `html` returns `undefined` if the selected element doesn't exist and just do `var friendsContent = $('#friends-msg').html()` and test that instead.